### ISSUE TITLE
Unable to read the 'paused' state of a AbstractMessageListenerContainer

### ISSUE THREAD

**Issue by patrickherrera** [spring-projects/spring-kafka#609]
Hi, I have a use case where I wish to pause consumption on a topic to allow a higher-priority topic to get ahead.

I can see that I can pause a listener container via the public pause() method and this seems to work, but for some reason the method to return the 'isPaused' state is protected.

Is there a reason why this is not public?  Could that be changed?

Thanks

**Comment by garyrussell** [comment]
Yes, we could make it public but it doesn't really reflect whether the container is actually paused yet, just that `pause()` has been called (it takes a little while before the actual pause occurs).

It that's all you need, it's a simple fix; if you actually need to know the actual state, it would take a bit more work.

We should also consider publishing events when each consumer thread actually pauses.

**Comment by garyrussell** [comment]
@patrickherrera The fix is on master and is available in 2.1.5.BUILD-SNAPSHOT.

We're currently planning on releasing 2.1.5.RELEASE next Wednesday.

**Comment by patrickherrera** [comment]
Thanks guys, fantastic work!  I'll check it out right now
